projects/ion/src/lib/popover/component/popover.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component, Input } from '@angular/core';
+import { Subject } from 'rxjs';
+import { SafeAny } from '../../utils/safe-any';
+
+import { PopoverPosition } from '../../core/types/popover';
+import { IconType, IonButtonProps } from '../../core/types';
+
+export interface PopoverProps {
+  ionPopoverTitle: string;
+  ionPopoverBody: SafeAny;
+  ionPopoverActions?: IonButtonProps[];
+  ionPopoverIcon?: IconType;
+  ionPopoverIconClose?: boolean;
+  ionPopoverPosition?: PopoverPosition;
+}
+
+@Component({
+  selector: 'ion-popover',
+  templateUrl: './popover.component.html',
+  styleUrls: ['./popover.component.scss'],
+  exportAs: 'PopoverComponent',
+})
+export class IonPopoverComponent {
+  @Input() ionPopoverTitle: string;
+  @Input() ionPopoverBody: SafeAny;
```

Comment:
On prototypes, I only saw popover body as a `string`. The `any` here is really necessary? You can check with designers to confirm if popover body will be any content or just a string.

```suggestion
  @Input() ionPopoverBody: string;
```

Reply:
It will be declared as `ElementRef`, in order to receive a component or template.